icu/icu4c/source/common/unicode/uloc.h

```diff
@@ -525,6 +525,17 @@ U_CAPI const char* U_EXPORT2
 uloc_getISO3Language(const char* localeID);
 
 
+/**
+ * Gets the 3 letter ISO language code for the specified 2 letter language code.
+ *
+ * @param langCode Iso639LanguageTwoLetterName
+ * @return language the ISO language code for langCode
+ * @stable ICU 2.0
```

Comment:
please remove this line here and below, this is our own custom addition so we shouldn't be claiming the API is stable since ICU 2.0 😄 

I'd also add a comment somewhere to make clear this is our addition, e.g. `DOTNET CUSTOM API` or something like that.

Reply:
Thanks, updated :)